### PR TITLE
Added Toolbar in FeedbackSettings

### LINKED ISSUE #37: Missing ActionBar in FeedbackSettings
**Actual Behaviour** There is no action bar present in the activity.

**Expected Behaviour**
There should be an Actionbar/Toolbar

**Steps to reproduce it**
Build and run the app.
Go to Feedback Settings from the drop-down menu.

**Would you like to work on the issue?**
Yes, I would like to work on this issue.

### PR DESCRIPTION
**Pull request by jddeep**
Fixes #37 

**Changes**: Added the Toolbar as ActionBar.

**Screenshot/s for the changes**: 
screenshot_20190206-085208

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [ ] I have requested reviews from other members

### CODE REVIEW

**Comment by jddeep** [comment]
@CloudyPadmal could you please review and merge the PR. Cheers!

**Comment by jddeep** [comment]
@mariobehling @CloudyPadmal could you please review and merge the PR. Thanks.

**Comment by jddeep** [comment]
@mariobehling @CloudyPadmal kindly review these pending PRs. They have been sitting for a long time now. Thanks.

File: app/src/main/java/io/neurolab/settings/FeedbackSettings.java [hunk 1]
```diff
@@ -27,6 +30,14 @@ public FeedbackSettings() {
...
         super.onCreate(savedInstanceState);
         setContentView(R.layout.fragment_feedback_settings);
+        Toolbar toolbar = findViewById(R.id.feedback_toolbar);
+        FeedbackSettings activity = this;
+        //Even if the API level is less than 21 this will work due to
+        // its presenence in XML.
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            activity.setActionBar(toolbar);
+        }
+        activity.getActionBar().setTitle("Neuro Lab");
```
Comment (CloudyPadmal) [inline app/src/main/java/io/neurolab/settings/FeedbackSettings.java]: Don't use hard-coded resources

File: app/src/main/java/io/neurolab/settings/FeedbackSettings.java [hunk 2]
```diff
@@ -27,6 +30,14 @@ public FeedbackSettings() {
     protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.fragment_feedback_settings);
+        Toolbar toolbar = findViewById(R.id.feedback_toolbar);
+        FeedbackSettings activity = this;
+        //Even if the API level is less than 21 this will work due to
```
Comment (CloudyPadmal) [inline app/src/main/java/io/neurolab/settings/FeedbackSettings.java]: Remove these comments

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]

**Comment by jddeep** [comment]
@CloudyPadmal Required changes have been seen into. You can now review the PR. Cheers!

**Comment by jddeep** [comment]
@CloudyPadmal kindly look into the pending PRs and merge them moving ahead. Cheers!

**Review by CloudyPadmal — APPROVED** [review]

**Pull request merged**